### PR TITLE
Add w-footer.

### LINKED ISSUE #1372: Rebuild footer.
**Is your feature request related to a problem? Please describe.**
As part of the devsite migration, we need to recreate the footer in a way that doesn't rely on their CSS.

**Describe the solution you'd like**
Tidy up the footer markup and CSS. Ideally it should look the same.

### PR DESCRIPTION
**Pull request by robdodson**
Fixes #1372.

Note: If this is written content for the site please include the issue number from your [content proposal](https://github.com/GoogleChrome/web.dev/issues?q=is%3Aissue+is%3Aopen+label%3A%22content+proposal%22).

Changes proposed in this pull request:

- Adds `.w-footer` which is a simplified version of the devsite footer.
- Adds sourcemaps to `sass.sh`

### CODE REVIEW

File: src/site/_includes/partials/footer.njk [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<footer class="w-footer">
+  <nav class="w-footer__linkboxes">
+    <ul class="w-footer__linkboxes-list">
+      <li class="w-footer__linkbox">
+        <h3 class="w-footer__linkbox-heading">Contribute</h3>
+        <ul class="w-footer__linkbox-list">
+          <li class="w-footer__linkbox-item">
+            <a href="https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=bug&template=bug_report.md&title="
+              class="w-footer__linkbox-link
+            gc-analytics-event" data-category="Site-Wide Custom Events" data-label="Footer Link (index 1)">
```
Comment (samthor) [inline src/site/_includes/partials/footer.njk]: gc-analytics-event seems ignored, we just use data-category and data-label to trigger the event.

Maybe this is an omission in DevSite v2.

File: src/site/_includes/partials/footer.njk [hunk 2]
```diff
@@ -0,0 +1,128 @@
+<footer class="w-footer">
+  <nav class="w-footer__linkboxes">
+    <ul class="w-footer__linkboxes-list">
+      <li class="w-footer__linkbox">
+        <h3 class="w-footer__linkbox-heading">Contribute</h3>
+        <ul class="w-footer__linkbox-list">
+          <li class="w-footer__linkbox-item">
+            <a href="https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=bug&template=bug_report.md&title="
```
Comment (samthor) [inline src/site/_includes/partials/footer.njk]: `target="_blank" rel="noopener"` on all links?

File: src/site/_includes/partials/footer.njk [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      </ul>
+    </nav>
+    <nav class="w-footer__utility-nav">
+      <ul class="w-footer__utility-list">
+        <li class="w-footer__utility-item
+            ">
+          <a class="w-footer__utility-link
+            gc-analytics-event" href="https://policies.google.com/" data-category="Site-Wide Custom Events"
+            data-label="Footer Terms and Privacy link">
+            Terms & Privacy
```
Comment (samthor) [inline src/site/_includes/partials/footer.njk]: &amp;amp;

File: src/site/_includes/partials/footer.njk (hunk 2 above)
Comment (robdodson) [reply src/site/_includes/partials/footer.njk]: I think the devsite policy is not to use target=_blank.

> We recommend not using target="_blank" to force such links to open a new tab; developers know how to open new tabs if they want to, so we should respect their decisions and leave the choice to them. For more thoughts on use of target="_blank", read [Paul Ingraham's blog](http://www.paulingraham.com/target-blank-is-evil.html).

The linked post makes a decent argument so I'm inclined to keep them if that's ok.

**Pull request merged**